app/src/main/java/com/sci/recipeandroid/feature/onboarding/ui/fragment/OnBoardFragment.kt

```diff
@@ -0,0 +1,141 @@
+package com.sci.recipeandroid.feature.onboarding.ui.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.bumptech.glide.Glide
+import com.sci.recipeandroid.R
+import com.sci.recipeandroid.databinding.FragmentOnboardBinding
+import com.sci.recipeandroid.feature.onboarding.ui.model.OnBoardModel
+
+class OnBoardFragment: Fragment() {
+    private var binding: FragmentOnboardBinding? = null
+    private var onBoardList = mutableListOf<OnBoardModel>()
```

Comment:
try to use immutable list --> listOf() instead of mutableList one if it is not really need to mutate items in the list

Reply:
Update in e2ceaf1519d2ad526bf9d419752063a820c4ad13